common/scripted_rules/00_rules.txt

```diff
@@ -366,9 +366,8 @@ can_raid_across_water = {
 		culture = {
 			OR = {
 				# Warcraft
-				has_sea_invader_innovation = yes
-				
-				has_cultural_parameter = culture_can_raid_at_sea_even_if_feudal
+				culture_can_raid_at_sea_even_if_feudal_trigger = yes
+				heritage_gives_sea_raiding_trigger = yes
```

Comment:
Actually, I believe this trigger is redundant since `culture_can_raid_at_sea_even_if_feudal_trigger` returns true for these heritages. Try to remove it and check in-game.

Reply:
Good catch! Tested and fixed it.